HW1/task1.py

```diff
@@ -0,0 +1,11 @@
+# Задача 2: Найдите сумму цифр трехзначного числа.
+# *Пример:*
+# 123 -> 6 (1 + 2 + 3)
+# 100 -> 1 (1 + 0 + 0)
+
+print("Введите трехзначное число")
+a = int(input())
```

Comment:
Зачем использовать функцию `print` для вывода сообщения о вводе? Когда это самое сообщение, можно и нужно помещать внутрь круглых скобок в функции `input`.
```suggestion
a = int(input("Введите трехзначное число "))
```